views-core/src/main/java/io/micronaut/views/turbo/TurboStream.java

```diff
@@ -69,16 +73,31 @@ public final class TurboStream implements Renderable {
     @Nullable
     private final String targetCssQuerySelector;
 
+    /**
+     * request-id attribute, only relevant when action=refresh
+     */
+    @Nullable
+    private final String requestId;
+
+    /**
+     * Morph only the children of the element designated by the target dom id.
+     */
+    private final boolean childrenOnly;
+
     @Nullable
     private final Object template;
 
     TurboStream(@NonNull TurboStreamAction action,
```

Comment:
keep the old constructor and annotate it with `@Depreacated` and also add `@deprecated` javadoc indicating to use the new one.

Reply:
good catch, have added it back